webrtc/src/data_channel/mod.rs

```diff
@@ -535,16 +569,13 @@ impl RTCDataChannel {
     /// on_buffered_amount_low sets an event handler which is invoked when
     /// the number of bytes of outgoing data becomes lower than the
     /// buffered_amount_low_threshold.
-    pub async fn on_buffered_amount_low<F>(&self, f: F)
-    where
-        F: OnBufferedAmountLowFn + 'static,
-    {
+    pub async fn on_buffered_amount_low(&self, f: Box<dyn OnBufferedAmountLowFn>) {
```

Comment:
Why did we have to revert back to the Box variant here?

Reply:
Because I would like to switch to generics in all callback setters in one pull request, now it's not possible